test/unit/puppetlabs/services/jruby/jruby_testutils.clj

```diff
@@ -86,7 +86,9 @@
                    {:pool                 pool
                     :id                   1
                     :jruby-puppet         (create-mock-jruby-instance)
-                    :scripting-container  (ScriptingContainer. LocalContextScope/SINGLETHREAD)
+                    :scripting-container (doto (ScriptingContainer. LocalContextScope/SINGLETHREAD)
+                                               (jruby-core/prep-scripting-container ruby-load-path gem-home)
+                                               (.runScriptlet "require 'puppet/server/master'"))
```

Comment:
I don't think we want to do this here; @dankreek originally put this in because loading the puppet code into the JRuby containers makes the tests take about 10x as long.  If you need this for a specific test, it's probably best to just put the logic directly in that test namespace rather than in the utility function.